### PR TITLE
Prepare 0.0.121

### PR DESCRIPTION
**Pull request by arik-so**

### CODE REVIEW

File: ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift [hunk 1]
```diff
@@ -757,7 +757,13 @@ public class HumanObjectPeerTestInstance {
...
 
             let channelManagerConstructor = peer1.constructor!
-            let invoicePaymentResult = Bindings.payInvoice(invoice: invoice, retryStrategy: Bindings.Retry.initWithAttempts(a: 3), channelmanager: channelManagerConstructor.channelManager)
+            
+            let (paymentHash, recipientOnion, routeParameters) = Bindings.paymentParametersFromInvoice(invoice: invoice).getValue()!
+            let paymentId = invoice.paymentHash()!
+            
+            
+            let invoicePaymentResult = channelManagerConstructor.channelManager.sendPayment(paymentHash: paymentHash, recipientOnion: recipientOnion, paymentId: paymentId, routeParams: routeParameters, retryStrategy: Bindings.Retry.initWithAttempts(a: 3))
+            // let invoicePaymentResult = Bindings.payInvoice(invoice: invoice, retryStrategy: Bindings.Retry.initWithAttempts(a: 3), channelmanager: channelManagerConstructor.channelManager)
```
Comment (tnull) [inline ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift:766]: nit: Remove commented-out line.

File: ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift [hunk 2]
```diff
@@ -853,7 +859,9 @@ public class HumanObjectPeerTestInstance {
                 try! await Task.sleep(nanoseconds: 0_100_000_000)
             }
 
-            let invoicePayment = invoicePaymentResult.getValue()!
+            
+            // let invoicePayment = invoicePaymentResult.getValue()!
```
Comment (tnull) [inline ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift:863]: nit: Remove commented-out line.

File: ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift [hunk 3]
```diff
@@ -951,7 +959,8 @@ public class HumanObjectPeerTestInstance {
                 try! await Task.sleep(nanoseconds: 0_100_000_000)
             }
 
-            let invoicePayment = invoicePaymentResult.getValue()!
+            // let invoicePayment = invoicePaymentResult.getValue()!
```
Comment (tnull) [inline ci/LDKSwift/Tests/LDKSwiftTests/HumanObjectPeerTestInstance.swift:962]: nit: Remove commented-out line.

**Review by tnull — APPROVED** [review]
LGTM, mod 3 nits.

**Comment by arik-so** [comment]
Will address in follow-up commit, don't wanna restart the CI from scratch right now.

**Pull request merged**